### ISSUE TITLE
DotnetToolSettings.xml missing in 8.6.40 and 8.6.45

### ISSUE THREAD

**Issue by petermorlion** [SteveGilham/altcover#180]
I managed to get AltCover working (thanks for your work!), but the visualizer won't install. Running

```
dotnet tool install --global altcover.visualizer
```

produces the following error:

```
The settings file in the tool's NuGet package is invalid: Settings file 'DotnetToolSettings.xml' was not found in the package.
Tool 'altcover.visualizer' failed to install. Contact the tool author for assistance.
```

Both 8.6.45 and 8.6.40 didn't work. 8.6.14 successfully installed.

**Comment by SteveGilham** [comment]
Bug introduced when changing to net5.0 as a target.  I've deprecated the affected packages and will update.

**Comment by SteveGilham** [comment]
Should be fixed in [release 8.6.48](https://www.nuget.org/packages/altcover.visualizer/8.6.48).

**Comment by petermorlion** [comment]
Verified that this is fixed. Thanks.
